### PR TITLE
Drop version number from WAI-ARIA title and shortTitle

### PR DESCRIPTION
**Pull request by sideshowbarker**
The “1.2” version number in the WAI-ARIA title isn’t necessary — and is outdated as far as the current spec goes; https://w3c.github.io/aria/ now has “1.3”.

Also repeating the “WAI-ARIA” shortTitle in the title value sees unnecessary. In the case of MDN — which uses "${title} ${shortTitle}, it causes the spec reference to be rendered like this:

> Accessible Rich Internet Applications (WAI-ARIA) 1.2 (WAI-ARIA 1.2)

But after this change, the ARIA spec reference will be just:

> Accessible Rich Internet Applications (WAI-ARIA)

### CODE REVIEW

**Comment by sideshowbarker** [comment]
It’s unclear to me why the “has series titles that look consistent with spec titles” lint is failing. I don’t know how to fix it.

**Comment by tidoust** [comment]
> The “1.2” version number in the WAI-ARIA title isn’t necessary — and is outdated as far as the current spec goes; https://w3c.github.io/aria/ now has “1.3”.

The problem is that the `title` property is at the root level of the spec object and could either be interpreted as "title of the release spec" or "title of the nightly spec". So far, it is the title of the release version of the spec (which is at version 1.2), while you interpret it as the title of the nightly version. The proper solution would be to rather have `title` properties under `release` and `nightly` as discussed in #360, which we haven't done yet.

In the meantime, I'll push another PR to update the title of WAI-ARIA and drop the version number.

You updated `index.json` in the PR. That file is automatically generated from `specs.json`, so updates need to take place there (and `index.json` should not have been part of the PR). The title consistency test fails because you didn't update the series title in `index.json`, and the test warns you that the spec title and the series title look misaligned.

Also, the test that you updated merely checks the short title generation logic, but does not look into either `specs.json` or `index.json`. It shouldn't be changed.

> Also repeating the “WAI-ARIA” shortTitle in the title value sees unnecessary. In the case of MDN — which uses "${title} ${shortTitle}":

I'm not sure that `${title} (${shortTitle})` is a good pattern: the short title is only meant to convey a shorter version of the title for situations where you want to reference the spec in prose without all the shebang. In most cases, you'll either end up with:
1. a short title that is just the title because there is no meaningful way to shorten the title, leading to references such as "Compute Pressure API (Compute Pressure API)".
2. a short title that is just the title without generic terms, leading to references such as "Fetch Standard (Fetch)" or "CSS Namespaces Module Level 3 (CSS Namespaces 3)".

Neither strikes me as good, and I don't know how to compute a better `shortTitle` in these cases. It seems to me that for MDN to be able to use `${title} (${shortTitle})` as a pattern, the `shortTitle` property should only be set when it conveys an acronym or some sufficiently distinct abbreviation (e.g. "WAI-ARIA" or "WebRTC").

In other words, I'm not clear that the title really needs fixing in browser-specs. Wouldn't it be better to update MDN's reference pattern to `${title}` (and later on to `${nightly.title}` once we have it)?

**Comment by tidoust** [comment]
PR #518 takes care of dropping the version number. I'm closing this PR but happy to continue the discussion about the `shortTitle` property.

I guess the question boils down to: What does MDN hope to get from the `shortTitle` property? A biblio entry name?

**Comment by sideshowbarker** [comment]
> You updated `index.json` in the PR. That file is automatically generated from `specs.json`, so updates need to take place there (and `index.json` should not have been part of the PR).

oops — thanks, next time I’ll make sure I get that done right

> The title consistency test fails because you didn't update the series title in `index.json`, and the test warns you that the spec title and the series title look misaligned.

ah — Ok, thanks

> PR #518 takes care of dropping the version number.

👍 Thanks

**Comment by sideshowbarker** [comment]
cc @Elchi3 

> I'm not sure that `${title} (${shortTitle})` is a good pattern: the short title is only meant to convey a shorter version of the title for situations where you want to reference the spec in prose without all the shebang. In most cases, you'll either end up with:
> 
>     1. a short title that is just the title because there is no meaningful way to shorten the title, leading to references such as "Compute Pressure API (Compute Pressure API)".
> 
>     2. a short title that is just the title without generic terms, leading to references such as "Fetch Standard (Fetch)" or "CSS Namespaces Module Level 3 (CSS Namespaces 3)".
> 
> Neither strikes me as good, and I don't know how to compute a better `shortTitle` in these cases. It seems to me that for MDN to be able to use `${title} (${shortTitle})` as a pattern, the `shortTitle` property should only be set when it conveys an acronym or some sufficiently distinct abbreviation (e.g. "WAI-ARIA" or "WebRTC").
> 
> In other words, I'm not clear that the title really needs fixing in browser-specs. Wouldn't it be better to update MDN's reference pattern to `${title}` (and later on to `${nightly.title}` once we have it)?

I don’t have a strong opinion either way. On the face of it, though, it does seem to me that’d be better — but that said, there may be some good reason (specific cases) I don’t know of that motivated including `${shortTitle}` in the output. I think @Elchi3 is person who could say.

Certainly for the case of most specs, it seems to me kind of redundant or unnecessary to have the short title in output. But maybe I’ve not scrutinized enough cases.

> I guess the question boils down to: What does MDN hope to get from the `shortTitle` property? A biblio entry name?

I don’t know myself, but I think @Elchi3 could answer that with authority.

**Comment by Elchi3** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/@charset#specifications 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range#specifications

These indicate that `shortTitle` is often included in `title` already? Maybe that was't the case a few months ago when we started using browser-specs?

I think MDN could drop `shortTitle` and avoid running into problems like these.

**Comment by sideshowbarker** [comment]
> I think MDN could drop `shortTitle` and avoid running into problems like these.

👍 OK, opened https://github.com/mdn/yari/pull/5283
